components/script/dom/node.rs

```diff
@@ -464,30 +449,13 @@ impl Node {
     }
 
     pub fn dirty(&self, damage: NodeDamage) {
-        self.dirty_impl(damage, false)
-    }
-
-    pub fn dirty_impl(&self, damage: NodeDamage, force_ancestors: bool) {
-        // 0. Set version counter
-        self.rev_version();
```

Comment:
Where did this rev_version call go? This is the one tht invalidates HTML collections, and which explain t least a bunch of the failures, sorry for overlooking it.